pages/bellapps.md

```diff
@@ -1,4 +1,10 @@
 #BeLL-Apps: Communities and Nations
+
+##Objective
```

Comment:
Yeah I am aware, the reason there aren't spaces there is because in some cases, my rawgit wouldn't display properly if I did include the space.. That said I will fix the formatting on each page now.